src/main/resources/messages_en.properties

```diff
@@ -255,8 +255,8 @@ purl_server.error.api.purl.domain=PURL {0} retrieves no domain!
 
 purl_server.error.login.passwords_different=The entered passwords don`t fit together.
 purl_server.error.login.email=Error while sending e-mail!
-purl_server.error.validate.purl.create.path.empty=Path not set!
 purl_server.error.validate.purl.create_modify.path.exist=Path exist!
+purl_server.error.validate.purl.create.path.partial=Partial bereits vergeben.
```

Comment:
Path is part of another Partial PURL!

Hier verstecken sich außerdem noch ein paar nicht übersetzte Texte.
Außerdem einheitlich entscheiden für Punkt oder Ausrufezeichen am Ende der Fehlermeldung